main.py

```diff
@@ -0,0 +1,267 @@
+		self.x = x
+		self.y = y-height
+		#size of chasing cube
+		self.width = width
+		self.height = height
+		#constant velocity
+		self.constdx = constdx
+		self.dx = dx
+		self.shiftdx = shiftdx
+
+	def step(self):
+		self.x += self.constdx
+
+# classes for level objects
+class Ground(object):
+	def __init__(self, x = 0, y = 300, width = 2400, height = 180,dx=0,shiftdx=-1):
+		self.x = x
+		self.y = y
+		#size of ground
+		self.width = width
+		self.height = height
+		self.dx = dx
+		#shiftdx is for moving the ground when player moves
+		self.shiftdx = shiftdx
+
+class Platform(object):
+	def __init__(self, x=0,y=0,width = 100, height = 20, dx=0, shiftdx=-1):
+		self.x = x
+		self.y = y
+		self.width = width
+		self.height = height
+		self.dx = dx
+		#shiftdx is for moving the ground when player moves
+		self.shiftdx = shiftdx
+
+"""View classes"""
+class PlayerView(object):
+	def __init__(self, model):
+		self.model = model
+
+	def draw(self, surface):
+		model = self.model
+		# this takes (x,y,width,height)
+		pygame.draw.rect(surface,SLATEGRAY,(model.x,model.y,model.width,model.height))
+
+class PainTrainView(object):
+	def __init__(self, model):
+		self.model = model
+
+	def draw(self, surface):
+		model = self.model
+		# this takes (x,y,width,height)
+		pygame.draw.rect(surface,BLACK,(model.x,model.y,model.width,model.height))
+
+class GroundView(object):
+	def __init__(self, model):
+		self.model = model
+
+	def draw(self, surface):
+		model = self.model
+		# this takes (x,y,width,height)
+		pygame.draw.rect(surface,DIMGRAY,(model.x,model.y,model.width,model.height))
+
+class ObstacleView(object):
+	# can be used for any rectangular object
+	def __init__(self,model):
+		self.model = model
+
+	def draw(self,surface):
+		model = self.model
+		pygame.draw.rect(surface,BLACK,(model.x,model.y,model.width,model.height))
+
+"""Controller classes"""
+class Controller(object):
+	def __init__(self,models):
+		self.models = models
+		self.player = models[0] # make sure this aligns with controlled_models in main
+
+	def handle_event(self):
+		# time passed isn't actually time based... based on while loop efficiency
+		player = self.player
+		models = self.models
+		keys = pygame.key.get_pressed() # checking pressed keys
+		for model in models:
+			#movements of player
+			if keys[pygame.K_LEFT]:
+				if player.go_back():
+					model.x -= model.shiftdx
+				else:
+					model.x -= model.dx
+			if keys[pygame.K_RIGHT]:
+				if player.shift_world():
+					model.x += model.shiftdx
+				else:
+					model.x += model.dx
+		if keys[pygame.K_UP] and player.dy == 0:
+			player.dy = player.jumpdy
+
+def main():
+	pygame.init()
+	screen = pygame.display.set_mode((640,480))
+
+	# models
+	# level models:
+	ground = Ground(x=0,width=600)
+	#5 different platforms
+	platform1 = Platform(10,10)
+	platform2 = Platform(800,10)
+	platform3 = Platform(1600,10)
+	platform4 = Platform(2000,10)
+	platform5 = Platform(2400,10)
+	# player/NPC models:
+	player = Player(300,300)
```

Comment:
Maybe use constants here. Especially a constant for the height of the ground, so if you wanted the change the starting height of the player and the pain train, you would just have to change one constant instead of 2 numbers.